libs/platform/src/lib/components/form/collection-base.input.ts

```diff
@@ -75,4 +111,38 @@ export abstract class CollectionBaseInput extends BaseInput {
             return this.objectGet(obj[is[0]], is.slice(1));
         }
     }
+
+    protected _setFieldLayout(): void {
+        try {
+            this._xlIsInline = this.inlineLayout['XL'];
+            this._lgIsInline = this.inlineLayout['L'];
+            this._mdIsInline = this.inlineLayout['M'];
+            this._sIsInline = this.inlineLayout['S'];
+            this._updateLayout();
+        } catch (error) {
+            this._isInLineLayoutEnabled = false;
+        }
+
+        if (this._isInLineLayoutEnabled) {
+            fromEvent(window, 'resize')
+                .pipe(debounceTime(50), takeUntil(this._destroyed))
+                .subscribe(() => this._updateLayout());
+        }
+    }
+
+    /** @hidden */
+    private _updateLayout(): void {
+        const width = window.innerWidth;
+
+        // check if value has changed, then only assign new value.
+        if (width > 0 && width < 600 && this.isInline !== this._sIsInline) {
```

Comment:
I would move these breakpoint sizes to the enum like:
```typescript
export enum RESPONSIVE_BREAKPOINTS {
    SM: 600;
    MD: 1024;
    LG: 1440;
}
```

Reply:
using enum now.